src/lib/tree.typ

```diff
@@ -9,17 +9,37 @@
 
 #let typst-content = content
 
-/// Layout and render tree nodes
+/// Lays out and renders tree nodes.
```

Comment:
Can you add subheadings after the parameter list to talk about the anchors and styling for trees?

Reply:
I've moved all the node documentation into the doc-string. The `tree` library needs 
a rework, though.